exercises/sieve/package.yaml

```diff
@@ -0,0 +1,20 @@
+name: sieve
+
+dependencies:
+  - base
+
+library:
+  exposed-modules: Sieve
+  source-dirs: src
+  dependencies:
+    - vector
```

Comment:
This is a problem for which I have no good answer. Maybe this will force us to give more attention to the example solutions, as they will define the packages available by default.

It's a design flaw. My bad! :grin: